WcaOnRails/app/webpacker/components/Admin/EditPerson.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import SemanticDatepicker from 'react-semantic-ui-datepickers';
+import { Button, Form, Icon } from 'semantic-ui-react';
+import UserSearch from '../SearchWidget/UserSearch';
+import { fetchJsonOrError } from '../../lib/requests/fetchWithAuthenticityToken';
+import { usersApiUrl } from '../../lib/requests/routes.js.erb';
+
+const description = `
+  <p>
+  Choose 'Fix' if you want to replace a person's information in the database.
+  It will modify the Persons table accordingly and the Results table if the person's name is different.
+  This should be used to fix mistakes in the database.
+  </p>
+
+  <p>
+  Choose 'Update' if the person's name or country has been changed.
+  It will add a new entry in the Persons table and make it the current information for that person (subId=1)
+  but it will not modify the Results table so previous results keep the old name.
+  </p>
+`;
+
+function EditPerson({ countryList, genderList }) {
+  const [name, setName] = useState('');
+  const [representing, setRepresenting] = useState('');
+  const [gender, setGender] = useState('');
+  const [birthdate, setBirthdate] = useState('');
+  const [incorrectClaimCount, setIncorrectClaimCount] = useState(0);
+  const [personFound, setPersonFound] = useState(false);
+  return (
+    <>
+      {/* eslint-disable-next-line react/no-danger */}
+      <div dangerouslySetInnerHTML={{ __html: description }} />
```

Comment:
This should never be necessary. Even though this can't be exploited, the description text can just be put in the jsx.

Reply:
The description is actually having HTML stuffs like p tag. That's why using this. Please let me know if this can be changed without causing warning.

Reply:
```
return (
  <>
    <div>
      <p>The description.</p>
      <p>More text.</p>
    </div>
    ...
```